.cirrus.yml

```diff
@@ -101,7 +101,7 @@ task:
   only_if: $CIRRUS_TAG == ''
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
   osx_instance:
-    image: mojave-xcode-11.3-flutter
+    image: catalina-xcode-11.4.1-flutter
```

Comment:
Can also be `catalina-flutter` to always point to the latest one.

Reply:
I think we need to require manually version updating, otherwise we're not going to be able to attribute test failures to changes in the CI environment. FYI @Hixie

Reply:
Agreed. Can make a note in the upgrade run book to always test new images on this repo before changing `catalina-flutter` alias.